sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.features.inc

```diff
@@ -61,6 +68,94 @@ function dccr_ftr_workshop_flag_default_flags() {
+        {
+          "value" : "The workshop [message:field-workshop-node:title] is full.\\n\\nThanks.",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EThe workshop [message:field-workshop-node:title] is full.\\u003C\\/p\\u003E\\n\\u003Cp\\u003EThanks.\\u003C\\/p\\u003E\\n"
+        }
+      ],
+      "es" : [
+        {
+          "value" : "El taller [message:field-workshop-node:title] est\\u00e1 lleno",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EEl taller [message:field-workshop-node:title] est\\u00e1 lleno\\u003C\\/p\\u003E\\n"
+        },
+        {
+          "value" : "El taller [message:field-workshop-node:title] ha completado su cupo.\\r\\n\\r\\nGracias.",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EEl taller [message:field-workshop-node:title] ha completado su cupo.\\u003C\\/p\\u003E\\n\\u003Cp\\u003EGracias.\\u003C\\/p\\u003E\\n"
+        }
+      ]
+    },
+    "rdf_mapping" : []
+  }');
+  $items['user_application_workshop'] = entity_import('message_type', '{
+    "name" : "user_application_workshop",
+    "description" : "User application to workshop",
+    "argument_keys" : [],
+    "argument" : [],
+    "category" : "message_type",
+    "data" : {
+      "token options" : { "clear" : 0 },
+      "purge" : { "override" : 0, "enabled" : 0, "quota" : "", "days" : "" }
+    },
+    "language" : "en",
+    "arguments" : null,
+    "message_text" : {
+      "en" : [
+        {
+          "value" : "You are registered in the workshop  [message:field-workshop-node:title]",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EYou are registered in the workshop  [message:field-workshop-node:title]\\u003C\\/p\\u003E\\n"
+        },
+        {
+          "value" : "Dear [current-user:name] \\r\\nYou are registered in the workshop [message:field-workshop-node:title], gonna be the [message:field-workshop-node:field-workshop-date:value:custom:d\\/m\\/Y - g:i a]\\r\\n\\r\\nThanks.",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EDear [current-user:name]\\u003Cbr \\/\\u003E\\nYou are registered in the workshop [message:field-workshop-node:title], gonna be the [message:field-workshop-node:field-workshop-date:value:custom:d\\/m\\/Y - g:i a]\\u003C\\/p\\u003E\\n\\u003Cp\\u003EThanks.\\u003C\\/p\\u003E\\n"
+        }
+      ],
+      "es" : [
+        {
+          "value" : "Usted se ha registrado en el taller de [message:field-workshop-node:title]\\n",
+          "format" : "plain_text",
+          "safe_value" : "\\u003Cp\\u003EUsted se ha registrado en el taller de [message:field-workshop-node:title]\\u003C\\/p\\u003E\\n"
+        },
+        {
+          "value" : "Querido [current-user:name]\\nUsted se ha registrado correctamente en el taller  [message:field-workshop-node:title] , sera el [message:field-workshop-node:field-workshop-date:value:custom:d\\/m\\/Y - g:i a]\\nGracias.\\n",
```

Comment:
You are registered in the workshop [message:field-workshop-node:title]; it's going to be on [message:field-workshop-node:field-workshop-date:value:custom:m\\/d\\/Y - g:i a]\\r\\n\\r\\nThanks.",